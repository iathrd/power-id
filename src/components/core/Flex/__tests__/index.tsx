import renderer from "react-test-renderer";
import { ThemeProvider } from "@mui/material";
import Component from "../index";
import { mocks } from "@utils/index";

const themeMock = mocks.getThemeMock();

it("renders correctly", () => {
  const tree = renderer
    .create(
      <ThemeProvider theme={themeMock}>
        <Component />
      </ThemeProvider>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
