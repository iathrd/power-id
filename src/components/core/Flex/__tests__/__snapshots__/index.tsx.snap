// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
Array [
  <style
    dangerouslySetInnerHTML={
      Object {
        "__html": ".css-12cr3xa{-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-flex-direction:row;-ms-flex-direction:row;flex-direction:row;-webkit-box-pack:center;-ms-flex-pack:center;-webkit-justify-content:center;justify-content:center;}",
      }
    }
    data-emotion="css 12cr3xa"
  />,
  <div
    className="MuiBox-root css-12cr3xa"
  />,
]
`;
